setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='django-freeze',
-    packages=['freeze'],
+    packages=['freeze', 'freeze.management', 'freeze.management.commands'],
```

Comment:
Undo this change since it is not necessary, please.

Reply:
It is, the `packages` parameter needs a full list, there is no automatic recursion.
It probably seems to work now because the python files are included as package data.